.travis.yml

```diff
@@ -74,7 +68,8 @@ script:
     - mkdir build && cd build
 
     - if [ "$FLAVOR" == "cs" ]; then cmake -DBOND_SKIP_GBC_TESTS=TRUE -DBOND_SKIP_CORE_TESTS=TRUE -DBOND_ENABLE_COMM=FALSE -DBOND_ENABLE_GRPC=FALSE ..; fi
-    - if [ "$FLAVOR" == "cs" ]; then make --jobs 2 DESTDIR=$HOME install; fi
+    - if [ "$FLAVOR" == "cs" ]; then make gbc; fi
+    - if [ "$FLAVOR" == "cs" ]; then make DESTDIR=$HOME install; fi
```

Comment:
If we didn't figure out why splitting this up helped, I'm okay with committing it as-is, but could you make an issue to investigate?